__tests__/utils/flattenTokens.test.js

```diff
@@ -67,5 +67,29 @@ describe('utils', () => {
       const sortedRet = ret.sort(sortBy('value'));
       expect(sortedRet).to.eql(sortedExpectedRet);
     });
+
+    it('should return nested leaf node values using DTCG spec as an array', () => {
```

Comment:
Was trying to reproduce something a user told me, didn't manage to but I figured I might as well leave the test here, nice to check that it works for DTCG tokens and up our coverage slightly.